### PR TITLE
Address Update Bulk Processor

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We want to be able to handle address updates from Address Resolution (AR) in bulk via files uploaded to a bucket.

# What has changed
Created a bulk processor for address updates.

# How to test?
Upload files with the prefix of `address_updates_` to the correct bucket.

# Links
Trello: https://trello.com/c/u0QkoepB

### CODE REVIEW

File: toolbox/bulk_processing/validators.py [hunk 1]
```diff
@@ -164,3 +172,59 @@ def validate(value, **_kwargs):
...
+def mandatory_after_update(column_name, label):
+    """Field must be present either on the case or the update row"""
+
+    def validate(value, **kwargs):
+        if value:
+            return
+
+        case_id = kwargs['row']['CASE_ID']
+        try:
+            case = execute_in_connection_with_column_names("SELECT * FROM casev2.cases WHERE case_id = %s LIMIT 1",
```
Comment (NickGrantX) [inline toolbox/bulk_processing/validators.py]: `LIMIT 1` seems excessive when using the primary key

File: toolbox/bulk_processing/address_update_processor.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    if row['HTC_WILLINGNESS']:
+        event_message['payload']['rmCaseUpdated']['htcWillingness'] = row['HTC_WILLINGNESS']
+    if row['HTC_DIGITAL']:
+        event_message['payload']['rmCaseUpdated']['htcDigital'] = row['HTC_DIGITAL']
+    if row['CE_EXPECTED_CAPACITY']:
+        event_message['payload']['rmCaseUpdated']['ceExpectedCapacity'] = row['CE_EXPECTED_CAPACITY']
+
+    if row['CE_SECURE'] == '0':
+        event_message['payload']['rmCaseUpdated']['secureEstablishment'] = False
+    if row['CE_SECURE'] == '1':
```
Comment (NickGrantX) [inline toolbox/bulk_processing/address_update_processor.py]:
```suggestion
    elif row['CE_SECURE'] == '1':
```

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Should update the readme for the new processor

File: toolbox/bulk_processing/address_update_processor.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+def is_delete_keyword(value: str):
+    return value == '-----'
+
+
+def main():
+    BulkProcessor(AddressUpdateProcessor()).run()
+
+
+if __name__ == '__main__':
```
Comment (ryangrundy7) [inline toolbox/bulk_processing/address_update_processor.py:159]: Probably worth adding an alias for this file

File: toolbox/bulk_processing/validators.py [hunk 4]
```diff
@@ -216,7 +216,7 @@ def validate(value: str, **_kwargs):
 
 
 def check_no_dodgy_hyphen_lookalikes(value):
-    for hyphen_lookalike in ('‒', '–', '—', '―', '_', '~', '¯', 'ˉ', 'ˍ', '˗', '‐', '‑', '‒', '‾', '⁃', '⁻', '₋', '−',
```
Comment (liamtoozer) [inline toolbox/bulk_processing/validators.py]: Did we want to keep tilde just in case?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

File: toolbox/bulk_processing/validators.py (hunk 4 above)
Comment (NickGrantX) [reply toolbox/bulk_processing/validators.py]: isn't going to hurt I don't think

File: toolbox/bulk_processing/address_update_processor.py (hunk 3 above)
Comment (NickGrantX) [reply toolbox/bulk_processing/address_update_processor.py:159]: done

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**